fpga_interchange/fasm_generators/generic.py

```diff
@@ -368,6 +377,10 @@ def fill_pip_features(self, pip_feature_format, extra_pip_features,
                                    name for p in pcell.pins):
                                 pin = bel_pin.name
 
+                        if pin == None:
+                            # TODO: GND driver has no input pin
+                            continue
+
                         assert pin
```

Comment:
Maybe worth to `continue` if the bel does not have input pins instead of continuing if the pin is `None`, so that the assertion is kept for cases in which a "normal" BEL does not produce any valid results.

Reply:
I agree this needs to be structured differently, but the problem is not the bel itself having no input pins, but the pseudo-pip specifying no input pins because a LUT is being used as a ground driver.

Reply:
Hmm, right, I think that, if this concerns only the LUT-as-GND-generator case, we might apply the `continue` only in cases where the BEL is a LUT element, so that at least the below assertion is still valid for routing BELs

Reply:
Agreed, done.